content/kubecarrier/master/getting_started/api_access.md

```diff
@@ -0,0 +1,230 @@
+---
+title: API Access
+weight: 70
+slug: api_access
+date: 2020-06-26T09:00:00+02:00
+---
+
+## Intro
+
+KubeCarrier also deploys it's own API Server to allow external access and integrations to connect with KubeCarrier.
+It's designed as a slim interface layer and all the heavy lifting (validation, authorization, etc.) is still done by Kubernetes Controllers, Kubernetes Admission Webhooks, and other Kubernetes mechanisms.
+
+We decided to build our own custom API Server for a few reasons:
+
+1. Exposing `kube-apiserver` on the public internet is discouraged.
+2. We want to enable separate authentication configuration for users of the KubeCarrier API.
+3. Long term, we want to support aggregation and discovery from multiple KubeCarrier clusters to enable advanced distributed deployments.
+
+## Testing the API
+
+The KubeCarrier API Server exposes a Open API Specification under `<host>/v1/openapi` and the Swagger UI under `<host>/v1/swagger/` to browse the API specification.
+
+To access the KubeCarrier API directly, you can use `kubectl port-forward` to expose the api server on localhost.
+
+First you need to get the name of the api server pod:
+
+```bash
+kubectl get po -n "kubecarrier-system"
```

Comment:
I will never understand why people are so keen on abbreviating "pod" to "po", especially in documentation and examples...